website/docs/reference/resource-configs/bigquery-configs.md

```diff
@@ -898,9 +898,13 @@ Find more information about materialized view limitations in Google's BigQuery [
 The BigQuery adapter supports Python models with the following additional configuration parameters:
 
 | Parameter               | Type        | Required | Default   | Valid values     |
-|-------------------------|-------------|----------|-----------|------------------|
+| :---------------------- | :---------- | :------- | :-------- | :--------------- |
 | `enable_list_inference` | `<boolean>` | no       | `True`    | `True`, `False`  |
 | `intermediate_format`   | `<string>`  | no       | `parquet` | `parquet`, `orc` |
+| `submission_method`     | `<string>`  | no       | ``        | `serverless`, `bigframes` |
```

Comment:
`cluster` is also a valid value.

Reply:
Can you help me fill this out? Is it a cluster id (and thus a string?)? Optional?

Reply:
```suggestion
| `submission_method`     | `<string>`  | no       | ``        | `serverless`, `bigframes`, `cluster` |
```